civihr_employee_portal/templates/smarty/notifications/html.tpl

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
```

Comment:
Where did you get the code for this?

Reply:
@mickadoo from another smarty template that is used to send notifications.